src/main/scala/ba/sake/hepek/bulma/component/BreadcrumbComponents.scala

```diff
@@ -1,24 +1,27 @@
 package ba.sake.hepek.bulma.component
 
 import ba.sake.hepek.bulma._
-import scalatags.Text.all._
+import scalatags.Text
+import scalatags.Text.all.{content, _}
 
 object BreadcrumbComponents extends BreadcrumbComponents
 
+case class BreadcrumbItem(label: String) extends BulmaElement {
+  override def content: Text.all.Frag = li(a(label))
```

Comment:
Are we supposed to be able to set where that `a`nchor goes?

Reply:
Yes, you're right. Maybe it'd be more complete in that way!